.coderabbit.yaml

```
@@ -0,0 +1,141 @@
+    - "[WIP]"
+    - "draft"
+    - "Release"
+    - "release"
+  drafts: false
+  base_branches:
+    - "main"
+    - "release/v1"
+  incremental_review: true
+
+chat:
+  auto_reply: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)